gen/cheader.tmpl

```diff
@@ -0,0 +1,234 @@
+
+{{if eq .Name "webgpu"}}
+#include <stdint.h>
+#include <stddef.h>{{"\n" -}}
+{{- else}}
+#include "webgpu.h"{{"\n" -}}
+{{- end}}
+
+{{- if .Constants}}
+{{- range .Constants}}
+{{- MComment .Doc 0}}
+#define WGPU_{{.Name | ConstantCase}} ({{.Value | CValue}})
+{{- end}}{{"\n" -}}
+{{- end}}
+
+{{- if eq .Name "webgpu"}}
+typedef uint32_t WGPUFlags;
+typedef uint32_t WGPUBool;{{"\n" -}}
+{{- end}}
+
+{{- if .Objects}}
+{{- range .Objects}}
+{{- if not .Fake}}
+typedef struct WGPU{{.Name | PascalCase}}Impl* WGPU{{.Name | PascalCase}} WGPU_OBJECT_ATTRIBUTE;
+{{- end}}
+{{- end}}{{"\n" -}}
+{{- end}}
+
+{{- if .Structs}}
+// Structure forward declarations{{"" -}}
+{{range .Structs}}
+struct WGPU{{.Name | PascalCase}};
+{{- end}}{{"\n" -}}
+{{- end}}
+
+{{- range $_, $entry := .Enums}}
+{{- MComment .Doc 0}}
+typedef enum WGPU{{.Name | PascalCase}} {
+    {{- range $entryIndex, $_ := .Entries}}
+    {{- MComment .Doc 4}}
+    {{- $value := 0 -}}
+    {{- if eq .Value "" -}}
+        {{- $value = $entryIndex -}}
+    {{- else -}}
+        {{- $value = ParseUint .Value 16 -}}
+    {{- end -}}{{"\n" -}}
+    {{printf "    WGPU%s_%s = %s%.4X,"  ($entry.Name | PascalCase) (.Name | PascalCase) $.EnumPrefix $value -}}
+    {{- end}}
+    WGPU{{.Name | PascalCase}}_Force32 = 0x7FFFFFFF
+} WGPU{{.Name | PascalCase}} WGPU_ENUM_ATTRIBUTE;{{"\n"}}
+{{- end}}
+
+{{- range $_, $entry := .Bitflags}}
+{{- MComment .Doc 0}}
+typedef enum WGPU{{.Name | PascalCase}} {
+    {{- range $entryIndex, $_ := .Entries}}
+        {{- MComment .Doc 4}}
+        {{- $value := "" -}}
+        {{- $valueCombination := .ValueCombination -}}
+        {{- range $valueIndex, $v := .ValueCombination -}}
+            {{- $v = (printf "WGPU%s_%s" ($entry.Name | PascalCase) ($v | PascalCase)) -}}
+            {{- if IsLast $valueIndex $valueCombination -}}
+                {{- $value = (print $value $v) -}}
+            {{- else -}}
+                {{- $value = (print $value $v " | ") -}}
+            {{- end -}}
+        {{- else -}}
+            {{- if eq .Value ""}}
+                {{- $value = (printf "0x%.8X" (BitFlagValue $entryIndex)) -}}
+            {{- else}}
+                {{- $value = (printf "0x%.8X" (ParseUint .Value 64)) -}}
+            {{- end}}
+        {{- end -}}{{"\n" -}}
+        {{printf "    WGPU%s_%s = %s,"  ($entry.Name | PascalCase) (.Name | PascalCase) $value -}}
+    {{- end}}
+    WGPU{{.Name | PascalCase}}_Force32 = 0x7FFFFFFF
+} WGPU{{.Name | PascalCase}} WGPU_ENUM_ATTRIBUTE;
+typedef WGPUFlags WGPU{{.Name | PascalCase}}Flags WGPU_ENUM_ATTRIBUTE;{{"\n"}}
+{{- end}}
+
+{{- if eq .Name "webgpu"}}
+typedef void (*WGPUProc)(void) WGPU_FUNCTION_ATTRIBUTE;{{"" -}}
+{{- end}}
+
+{{- if .FunctionTypes}}
+{{- range .FunctionTypes}}
+{{- MComment .Doc 0}}
+typedef {{FunctionReturns .}} (*WGPU{{.Name | PascalCase}})({{FunctionArgs . nil}}) WGPU_FUNCTION_ATTRIBUTE;
+{{- end}}{{"" -}}
+{{- end}}
+
+{{- if .Objects}}
+{{- range $_, $object := .Objects}}
```

Comment:
```suggestion
{{- range $object := .Objects}}
```

here and below. If you accept this, you can fix all the occurrences with `{{- range $_,` -> `{{- range `